index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Skeleton</title>
+
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="logo"><h2>My logo</h2></div>
+        <div class="menu"><img src="images/Enabled.png" alt="" /></div>
+      </nav>
+    </header>
+    <main>
+      <section class="hello">
+        <div class="hellosection">
+          <div class="box box1">I’m Mariane Glad to see you!</div>
+          <div class="box box2">
+            I’m a software developer! I can help you build a product , feature
+            or website Look through some of my work and experience! If you like
+            what you see and have a project you need coded, don’t hestiate to
+            contact me.
+          </div>
+          <div class="box box3">
+            <div>LET'S CONNECT</div>
+            <div>
```

Comment:
- [ ] Please, use for "LET`S CONNECT" subtitle a header tag `hx` (examples: h1, h2 ,h3, etc...)

Reply:
h3 tag added for LET'S CONNECT subtitle